src/components/ContactInfo/index.jsx

```diff
@@ -3,10 +3,11 @@ import cn from 'classnames';
 import { charityAPI } from '../../clients';
```

Comment:
Please use `useCharityAPI` hook. 
even if you have two API calls for this component, you still can use the `useCharityAPI` hooks twice ;)